pyproject.toml

```diff
@@ -19,6 +19,7 @@ social-auth-app-django  = "3.1.0"
 social-auth-core = "3.2.0"
 herbie-json-schema = { git = "https://github.com/project-a/herbie-json-schema.git", branch = "master" }
 google-cloud-pubsub = "^1.1.0"
+strict-rfc3339 = "^0.7"
```

Comment:
I have no idea what pyproject.toml is good for … but i added the library also under `.dependencies` because poetry complained about it …